tests/load/snowflake/test_snowflake_configuration.py

```diff
@@ -103,12 +120,14 @@ def test_snowflake_credentials_native_value(environment) -> None:
         )
     # set password via env
     os.environ["CREDENTIALS__PASSWORD"] = "pass"
+    os.environ["CREDENTIALS__APPLICATION"] = "dlt"
     c = resolve_configuration(
         SnowflakeCredentials(),
         explicit_value="snowflake://user1@host1/db1?warehouse=warehouse1&role=role1",
     )
     assert c.is_resolved()
     assert c.password == "pass"
+    assert "application=dlt" in str(c.to_url())
```

Comment:
Could you put a test for the exact `dltHub_dlt` string? Because we'll be submitting it and we need to be sure it's the same

Reply:
We are checking the default at https://github.com/dlt-hub/dlt/pull/1266/files#diff-7d1eaf8541220ca58e41bdbb425431e2930d50076f8ac88f5282800ae1be696fR82